tests/ignite/trainer/test_trainer.py

```diff
@@ -274,19 +264,17 @@ def end_of_iteration_handler(trainer):
 def test_terminate_after_training_iteration_skips_validation_run():
```

Comment:
I guess this test can be removed as we now test above that `test_validate_is_not_called_by_default`